config/configtelemetry/configtelemetry.go

```diff
@@ -71,16 +66,16 @@ func (l *Level) UnmarshalText(text []byte) error {
 
 	str := strings.ToLower(string(text))
 	switch str {
-	case levelNoneStr:
+	case strings.ToLower(LevelNone.String()):
 		*l = LevelNone
 		return nil
-	case levelBasicStr:
+	case strings.ToLower(LevelBasic.String()):
 		*l = LevelBasic
 		return nil
-	case levelNormalStr:
+	case strings.ToLower(LevelNormal.String()):
 		*l = LevelNormal
 		return nil
-	case levelDetailedStr:
+	case strings.ToLower(LevelDetailed.String()):
```

Comment:
I know that it is not performance critical this code, but does not seem nice to call toLower every time, also call String which is another switch every time...

Maybe keep the constants as capitalized at least and here you have `strings.ToLower(levelNoneStr)`?

Reply:
> Maybe keep the constants as capitalized at least and here you have strings.ToLower(levelNoneStr)?

Ok, applied this suggestion